test/CMakeLists.txt

```diff
@@ -1,408 +1,90 @@
-  add_custom_command(OUTPUT "${EC64}.cpp"
-                     DEPENDS "${EC64}.bc"
-                     COMMAND binary2cpp external_code_extern_bitcode_64 < "${EC64}.bc" > "${EC64}.cpp")
-
-  set(ECCPP "${PROJECT_BINARY_DIR}/${PROJECT_NAME}.build/external_code_extern_cpp_source")
-  add_custom_command(OUTPUT "${ECCPP}.cpp"
-                     DEPENDS "${GEN_TEST_DIR}/external_code_extern.cpp"
-                     COMMAND binary2cpp external_code_extern_cpp_source < "${GEN_TEST_DIR}/external_code_extern.cpp" > "${ECCPP}.cpp")
-
-  add_library(external_code_generator_deps "${EC32}.cpp" "${EC64}.cpp" "${ECCPP}.cpp")
-
-  # ------ Generator tests for just-in-time mode: ------
-  # Find all the files of form "foo_jittest.cpp" in test/generator
-  # and declare a rule for them, each of which depends on foo.generator.
-  file(GLOB JITTEST_SRCS RELATIVE "${GEN_TEST_DIR}" "${GEN_TEST_DIR}/*_jittest.cpp")
-  foreach(FILE ${JITTEST_SRCS})
-    string(REPLACE "_jittest.cpp" "" NAME "${FILE}")
-    set(TARGET "generator_jit_${NAME}")
-    halide_project("${TARGET}" "generator" "${GEN_TEST_DIR}/${FILE}")
-    target_link_libraries("${TARGET}" PRIVATE "${NAME}.generator")
-    add_halide_test("${TARGET}"
-             GROUPS test_generator run_tests)
-  endforeach()
-
-  # ------ Generator tests for ahead-of-time mode: ------
-  # Find all the files of form "foo_aottest.cpp" in test/generator
-  # and declare a rule for them, each of which depends on foo.generator.
-  # Note that there are many special cases.
-  # TODO(srj)
-  # file(GLOB JITTEST_SRCS RELATIVE "${GEN_TEST_DIR}" "${GEN_TEST_DIR}/*_aottest.cpp")
-  # foreach(FILE ${JITTEST_SRCS})
-  #   string(REPLACE "_aottest.cpp" "" NAME "${FILE}")
-  #   set(TARGET )
-
-  #   add_executable("generator_aot_${NAME}" "${GEN_TEST_DIR}/${FILE}")
-  #   target_include_directories("generator_aot_${NAME}" PRIVATE
-  #                              "${CMAKE_SOURCE_DIR}"
-  #                              "${CMAKE_SOURCE_DIR}/tools"
-  #                              "${CMAKE_SOURCE_DIR}/src/runtime")
-
-  #   if (NOT ${args_OMIT_DEFAULT_GENERATOR})
-  #     halide_library_from_generator("${NAME}"
-  #                            GENERATOR "${NAME}.generator"
-  #                            FUNCTION_NAME "${args_FUNCTION_NAME}"
-  #                            HALIDE_TARGET "${args_GENERATOR_HALIDE_TARGET}"
-  #                            GENERATOR_ARGS "${args_GENERATOR_ARGS}"
-  #                            FILTER_DEPS "${args_FILTER_DEPS}")
-  #     target_link_libraries("generator_aot_${NAME}" PUBLIC "${NAME}")
-  #   endif()
-  # endforeach()
-
-  # Create tests for ahead of-time-compiled generators. This will produce two
-  # executables, one containing the generator itself
-  # (e.g. from example_generator.cpp) and used at build time, and the other, the
-  # test that executes the generated code (e.g. from example_aottest.cpp).
```

Comment:
@steven-johnson This was commented out to begin with.

Reply:
ah, ok